types/wlr_output.c

```diff
@@ -455,3 +455,57 @@ uint32_t wlr_output_get_gamma_size(struct wlr_output *output) {
 	}
 	return output->impl->get_gamma_size(output);
 }
+
+void wlr_output_transform_apply_to_box(enum wl_output_transform transform,
```

Comment:
Probably goes in `wlr_box.h` as `wlr_box_transform()`

Reply:
I'm not sure about it. `enum wl_output_transform` is specific to outputs so it makes sense to put it here, and to expose it only to backends (via `wlr/interfaces/wlr_output.h`). Also, I've added another helper `wlr_output_transformation_invert` that wouldn't fit in `wlr_box.h`.

Reply:
wl_output_transform comes from wayland, not wlroots. We can use it wherever we want.

Reply:
Sure. But these functions don't make sense outside backends - in particular, the compositor shouldn't use it.

Reply:
Alright.